source/BoardingPanel.cpp

```diff
@@ -396,6 +416,14 @@ bool BoardingPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command,
 					}
 				isCapturing = false;
 				
+				// You need to split the value of the ship with your crew.  You
+				// get 1000 shares, each crew member gets one.  If you don't
+				// pay, surviving crewmembers demand a higher interest rate
+				// than the familes of dead crewmembers.
+				int64_t reparations = (victim->Cost() * casualties * Depreciation::Full()) / (yourStartCrew + 999);
+				deathBenefits += reparations;
+				int64_t bonus = (victim->Cost() * (you->Crew() - 1) * Depreciation::Full()) / (yourStartCrew + 999);
+				profitShares += bonus;
```

Comment:
This is a good start, but the calculation that we'll actually need is more complex than this.

Have a look at `Crew.h` and `Crew.cpp`; that's where we currently keep this kind of logic. You may be able to reuse some of the existing functionality.

Reply:
Yep, that's on the TODO!